app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb

```diff
@@ -272,7 +272,7 @@ def parse_physical_server(node)
         :type                   => "ManageIQ::Providers::Lenovo::PhysicalInfraManager::PhysicalServer",
         :name                   => node.name,
         :ems_ref                => node.uuid,
-        :uid_ems                => @ems.uid_ems,
+        :uid_ems                => node.uuid,
```

Comment:
We shouldn't have two variables with the same value. `uid_ems` was never used, I believe we can remove this.

What's `@ems.uid_ems` represents @juliancheal ?

Reply:
@AndreyMenezes  according to the file `models/manager_refresh/inventory/core.rb` at line `62`, and some converstations that I had with some RH guys, the `uid_ems` is the reference of the manager in the database.

Reply:
It depends on the model, the idea is that ems_ref is how the provider references the object and uid_ems is something we can use to reference the object across providers.  It is okay if they are the same value as long as that value fills both roles.

The reference to the manager is ems_id.

Reply:
Got it @agrare it was the guys passed to me. In this case, I think it's LGTM.

Reply:
@agrare If it's not necessary `uid_ems` to refer the provider, in this case I believe they could be the same value.